src/R/Components/Impl/PackageManager/Implementation/View/PackageSourcesOptionsControl.cs

```diff
@@ -0,0 +1,227 @@
+﻿using System;
+using System.Drawing;
+using System.Drawing.Drawing2D;
+using System.Drawing.Text;
+using System.Windows.Forms;
+using System.Windows.Forms.VisualStyles;
+using Microsoft.R.Components.PackageManager.Implementation.ViewModel;
+
+namespace Microsoft.R.Components.PackageManager.Implementation.View {
+    public partial class PackageSourcesOptionsControl : UserControl {
+        private Size _checkBoxSize;
+
+        public PackageSourcesOptionsControl() {
+            InitializeComponent();
+            UpdateDpi();
+        }
+
+        private void UpdateDpi() {
+            var imgs = images16px;
+            if (addButton.Height > 72) {
+                imgs = images64px;
+            } else if (addButton.Height > 40) {
+                imgs = images32px;
+            }
+
+            addButton.ImageList = imgs;
+            removeButton.ImageList = imgs;
+            MoveUpButton.ImageList = imgs;
+            MoveDownButton.ImageList = imgs;
+        }
+
+
+        private void PackageSourcesContextMenu_ItemClick(object sender, ToolStripItemClickedEventArgs e) {
+            throw new NotImplementedException();
```

Comment:
Do we then need to remove attribute from the package so Tools | Options page will be completely out?